packages/elements/src/overlay/managers/interaction-lock-manager.ts

```diff
@@ -181,8 +181,14 @@ export class ScrollLockManager {
       document.scrollingElement.scrollLeft = this.scrollLeft;
     } else {
       // Since we don't know if is the body or html, set both.
-      document.documentElement.scrollTop = document.body.scrollTop = this.scrollTop;
-      document.documentElement.scrollLeft = document.body.scrollLeft = this.scrollLeft;
+      if (document.documentElement) {
```

Comment:
I don't think we need to handle `document.documentElement` removal.

Reply:
Agreed and reverted.